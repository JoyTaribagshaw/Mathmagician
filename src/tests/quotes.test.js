import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quotes';

test('Snapshot test for Quote component', () => {
  const component = renderer.create(<Quote />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it('If it renders correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
