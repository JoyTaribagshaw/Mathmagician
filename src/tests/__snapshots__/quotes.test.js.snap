// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`If it renders correctly 1`] = `
<div
  className="loading-message"
>
  Loading quote, please wait...
</div>
`;

exports[`Snapshot test for Quote component 1`] = `
<div
  className="quote-container"
>
  <p
    className="quote"
  />
  <h2
    className="author"
  />
</div>
`;
